packages/api-client/src/views/Request/RequestSidebarItem.vue

```diff
@@ -373,6 +376,34 @@ function openCommandPaletteRequest() {
                 icon="Ellipses"
                 size="sm" />
             </ScalarButton>
+            <ScalarTooltip
+              v-if="item.watchForChanges"
+              side="bottom"
+              :sideOffset="7">
+              <template #trigger>
+                <ScalarIcon
+                  class="text-sm"
+                  :class="
+                    activeCollection?.uid === item.entity.uid
+                      ? 'text-c-1'
+                      : 'text-c-3'
+                  "
+                  icon="Watch"
+                  size="sm"
+                  thickness="2.5" />
+              </template>
+              <template #content>
+                <div
+                  class="grid gap-1.5 pointer-events-none max-w-[320px] w-content shadow-lg rounded bg-b-1 z-100 p-2 text-xxs leading-5 z-10 text-c-1">
+                  <div class="flex items-center text-c-2">
+                    <span class="text-pretty">
+                      Your Open API document is being monitored for changes to
```

Comment:
This could maybe be a little shorter? Could also be nice to include the URL here since I don't think we show it anywhere else

Maybe:

> Watching: https://galaxy.scalar.com/openapi.yaml

Reply:
great idea about displaying the url here 👍